arangod/Cluster/DBServerAgencySync.cpp

```diff
@@ -173,16 +200,17 @@ DBServerAgencySyncResult DBServerAgencySync::execute() {
   // It is crucial that the following happens before we do `getLocalCollections`!
```

Comment:
although it is not relevant to this PR it would be nice to give a _reason_ for why this is crucial.